src/servers/main.cc

```diff
@@ -82,6 +82,12 @@ int metrics_port_ = 8002;
 // Should GPU metrics be reported.
 bool allow_gpu_metrics_ = false;
 
+// The number of threads to initialize for handling GRPC infer requests.
+int grpc_infer_cnt_ = 1000;
```

Comment:
I think around 100 will be good. The thread will not be released until the request is completed, so the value for this will be a bit tricky. Say if the value is set to 16, and there is a big model with max batch size 32 and dynamic batching. If the client send 32 batch-1 requests, only every 16 requests will be batched and processed.

In the future, we will need to decouple the gRPC thread's life cycle a bit more so that it is released once the request is transferred to the scheduler and another thread will pick up if the response is ready.

Reply:
I see. Then leave them both at 1000 then since for some cases batching > 100 is used.